src/Spinner.js

```diff
@@ -4,12 +4,33 @@ import classNames from 'classnames';
 import { mapToCssModules, tagPropType } from './utils';
 
 const propTypes = {
+  /**
+   * Set a custom element for this component
+   */
```

Comment:
Since these are not adding any other jsdoc params, you can just type it as:
```suggestion
  /** Set a custom element for this component */
```